packages/webamp/js/reducers/tracks.ts

```diff
@@ -6,6 +6,27 @@ export interface TracksState {
   [id: string]: PlaylistTrack;
 }
 
+let finalKhz: String = 'NaN';
+let finalKbps: String = 'NaN';
+function massageKhz(khz: number) {
+  let stringKhz = String(Math.round(khz / 1000));
+  if (khz/1000 != null) finalKhz = stringKhz;
```

Comment:
Can we avoid doing this division inside each if? Just do it once at the top of the function?